demo/grafana/dashboards/dashboard.json

```diff
@@ -64,7 +64,7 @@
           "formattedQuery": "",
           "hide": false,
           "intervalFactor": 1,
-          "query": "SELECT\n$timeSeries as t,\navg(Value)\nFROM $table\nWHERE $timeFilter\nAND Path='my.awesome.path'\nGROUP BY t\nORDER BY t",
+          "query": "SELECT \n$timeSeries as t, \navg(Value) \nFROM graphite \nALL INNER JOIN \n( \nSELECT Path \nFROM graphite_tagged \nWHERE \nTag1 = 'a=1' \nor Tag1 = 'c=5' \nGROUP BY (Path, Tags) \nHAVING count(Tag1) == 2 \n) USING  Path \nWHERE $timeFilter \nGROUP BY t \nORDER BY t",
```

Comment:
Could you please add some indication of tags used - both for filtering and grouping. Also, while we don't have a CI for that, please, attach produced image in comments (not in the repo, just here).

Reply:
Do you want me to explain how this query works, or what is it doing, or smth else? 
These tags are meaningless now, of course.
[Image](https://imgur.com/AkzSJgG)